### DISCUSSION TITLE
Is it possible to use Named transformations with this module?

### DISCUSSION THREAD

**Discussion by christine927t** [nuxt-modules/cloudinary#251] [Q&A]
We've recently implemented Cloudinary's Named transformations to the images in our UI with the @nuxt/image module. This is my first time adding your module because we want to use CldProductGallery but I don't see how to add a named transformation to the mediaAssets object, only normal transformations. Can anyone let me know if this is possible please?

**Comment by Baroshem** [comment]
Hey there,

I checked the docs and I dont think named transformations are supported by the widget itself :(

**Reply by christine927t** [reply]
Ok thank you for getting back to me!
